osidb/filters.py

```diff
@@ -75,6 +75,8 @@ class FlawFilter(DistinctFilterSet):
     changed_before = DateTimeFilter(
         field_name="updated_dt", method="changed_before_filter"
     )
+    is_major_incident = BooleanFilter(method="is_major_incident_filter")
```

Comment:
Can't this filter be based on the `is_major_incident` field? Then you'd avoid a lot of duplication

Reply:
Do you mean `is_major_incident = BooleanFilter(field_name="is_major_incident")`?

It seems it is working. Interesting that we did not think of it earlier. Could you please approve that I get you right? If so, I will update the code.

Reply:
Yes that is what I meant, I think the confusion comes from the deprecation docs which stated that all usage of the field *must* be removed -- but that is incorrect, write operations on the field should be removed but read operations can be kept since the database field won't be accessed in any way.

I amended the documentation in the PR that introduces the custom `deprecate_field`.

Reply:
In the end, I cannot use it.
When trying `flaw__is_major_incident = BooleanFilter(field_name="flaw__is_major_incident")` in `AffectFilter`, I got `FieldError at /osidb/api/v1/affects, Related Field got invalid lookup: is_major_incident` when accessing `http://localhost:8000/osidb/api/v1/affects?flaw__is_major_incident=true`.

I would prefer to leave the code as it is now, so we do not end up in a rabbit hole.